DEVELOPMENT.md

```diff
@@ -213,3 +219,63 @@ will not be visible inside container. The driver does not detect the
 root cause of that problem during start-up, but only when a volume
 creation has failed. This problem can be avoided by specifying
 explicit mount of /dev in the PMEM-CSI manifest.
+
+Repository elements which are generated or created separately
+=============================================================
+
+Here are creation and update notes for these elements in the repository
+which are not hand-edited
+
+Top-level README diagrams describing LVM and Direct device modes
+----------------------------------------------------------------
+Two diagrams are created using the [_dia_ drawing program](https://en.wikipedia.org/wiki/Dia_(software)).
+The [single source file](/docs/diagrams/pmem-csi.dia) has
+layers: {common, lvm, direct} so that two diagram variants can be produced from single source.
+Image files are produced by saving in PNG format with correct set of layers visible.
+The PNG files are commited as repository elements in docs/images/devicemodes/.
+
+Top-level README diagram describing communication channels
+----------------------------------------------------------------
+This diagram was created using the [_dia_ drawing program](https://en.wikipedia.org/wiki/Dia_(software)).
+Image file is produced by saving in PNG format.
+The PNG file is commited as repository elements in docs/images/communication/.
+
+Diagrams describing provisioning sequence
+-----------------------------------------------------------------------
+Two diagrams are generated using [plantuml program](http://plantuml.com/).
+Source files:
+- [source file for regular sequence](/docs/diagrams/sequence.wsd)
+- [source file for cache sequence](/docs/diagrams/sequence-cache.wsd).
+
+The PNG files are commited as repository elements in docs/images/sequence/.
+
+protobuf-compiler
+-----------------
```

Comment:
Instead of "protobuf-compiler" section, It would be good if can have something like "RegistryServer spec" that explains how to regenerate the source from the spec.

Reply:
fixed